tools/FakeItEasy.Build/Program.cs

```diff
@@ -110,10 +111,23 @@ public static void Main(string[] args)
             RunTargets(args);
         }
 
-        private static void RunTests(string testDirectory)
+        private static IEnumerable<(string directory, string framework)> GetDirectoriesAndFrameworks(string[] directories) =>
+            from directory in directories from framework in GetFrameworks(directory) select (directory, framework);
+
+        private static string[] GetFrameworks(string directory)
+        {
+            var filename = $"{directory.Split('/').Last()}.{(directory.Contains("VB") ? "vbproj" : "csproj")}";
+
+            return File.ReadAllText($"{directory}/{filename}")
+                .Split("<TargetFrameworks>", 2)[1]
+                .Split("</TargetFrameworks>", 2)[0]
+                .Split(';');
+        }
+
+        private static void RunTests(string testDirectory, string framework)
         {
             var xml = Path.GetFullPath(Path.Combine(TestsDirectory, Path.GetFileName(testDirectory) + ".TestResults.xml"));

```

Comment:
It seems to me we losing the XML output for all but the last-run framework.

Reply:
Looks like you're right. Well spotted!
By default, if you don't specify the framework, `dotnet xunit` generates one result file per framework (based on the specified filename). But if you do specify the framework, it generates one file with the specified name.